starter_code/elevator.js

```diff
@@ -1,19 +1,39 @@
 class Elevator {
-  constructor(){
-    this.floor      = 0;
-    this.MAXFLOOR   = 10;
-    this.requests   = [];
+  constructor() {
+    this.floor = 0;
+    this.MAXFLOOR = 10;
+    this.requests = [];
+    this.direction = "";
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
-  _passengersLeave() { }
-  floorUp() { }
-  floorDown() { }
-  call() { }
-  log() { }
+  start() {
+    var interval = setInterval(() => {
+      this.update();
+    }, 1000);
+  }
+
+  stop() {
+    clearInterval(interval);
```

Comment:
`interval` is a local variable inside the scope of the `start` function, so you can't use it here. Instead, you should save the returned interval id that you got in the `start` method as a property, such as 
`this.intervalId =  setInterval(() => {
 this.update();
 }, 1000);`

Then, here you'd be able to invoke `clearInterval(this.intervalId)`